rubocop-rails_config.gemspec

```diff
@@ -12,8 +12,10 @@ Gem::Specification.new do |spec|
   spec.license               = "MIT"
   spec.required_ruby_version = ">= 2.3.0"
 
-  spec.add_dependency "rubocop", "~> 0.60"
+  spec.add_dependency "rubocop", "~> 0.70"
   spec.add_dependency "railties", ">= 3.0"
+  spec.add_dependency "rubocop-performance", "~> 1.3"
+  spec.add_dependency "rubocop-rails", "~> 2.0"
```

Comment:
This is a small point. It may be helpful to summarize the dependency on RuboCop.

```diff
- spec.add_dependency "rubocop", "~> 0.60"
+ spec.add_dependency "rubocop", "~> 0.70"
+ spec.add_dependency "rubocop-performance", "~> 1.3"
+ spec.add_dependency "rubocop-rails", "~> 2.0"
  spec.add_dependency "railties", ">= 3.0"
```

Reply:
yes, that's neater. Done.